basket-service/src/main/java/com/yandex/market/basketservice/controller/BasketController.java

```diff
@@ -0,0 +1,53 @@
+package com.yandex.market.basketservice.controller;
+
+import com.yandex.market.basketservice.dto.ProductResponseDto;
+import com.yandex.market.basketservice.dto.ProductRequestDto;
+import com.yandex.market.basketservice.service.BasketService;
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+import org.springframework.data.web.PageableDefault;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.*;
+
+import static com.yandex.market.util.HttpUtils.PUBLIC_API_V1;
+import static org.springframework.http.HttpStatus.*;
+
+@RestController
+@RequestMapping(PUBLIC_API_V1)
+@RequiredArgsConstructor
+public class BasketController {
+
+    private final BasketService basketService;
+
+    @GetMapping("/{userId}/basket/products")
```

Comment:
```suggestion
    @GetMapping("/users/{userId}/baskets")
```
Вообще надо подумать над эндпоинтами, это всегда боль
Но точно все надо во множественном числе писать

Reply:
"/users/{userId}/basket"
у юзера и баскета связь one to one